timer/src/test/java/com/toggl/timer/startedit/domain/DurationInputtedActionTests.kt

```diff
@@ -56,7 +58,7 @@ internal class DurationInputtedActionTests {
     }
 
     @Test
-    fun `when the editable time entry is running`() = runBlockingTest {
+    fun `sets its value to the editableTimeEntry's startTime when the time entry is running`() = runBlockingTest {
```

Comment:
`The DurationInputted action sets its value to the editableTimeEntry's startTime when the time entry is running`?
I think you meant 
```suggestion
    fun `sets the editableTimeEntry's startTime when the time entry is running`() = runBlockingTest {
```